dask/dataframe/core.py

```diff
@@ -1537,7 +1537,7 @@ def quantile(self, q=0.5, axis=0, method='default'):
 
     @derived_from(pd.DataFrame)
     def describe(self, split_every=False, percentiles=None, percentiles_method='default'):
-        # currently, only numeric describe is supported
+        """Currently, only numeric describe is supported """
```

Comment:
We might use this function as a test for the derived_from functionality change in this PR. 

As an example, I think that there is already a test in `test_utils.py` that import dask.dataframe and checks the state of one of the docstrings.

Reply:
good idea